src/geo/leaflet/leaflet-map-view.js

```diff
@@ -45,8 +45,15 @@ var LeafletMapView = MapView.extend({
       // remove the "powered by leaflet"
       this.map_leaflet.attributionControl.setPrefix('');
 
+      // Disable scrollwheel
       if (this.map.get("scrollwheel") == false) this.map_leaflet.scrollWheelZoom.disable();
+      // Disable keyboard
       if (this.map.get("keyboard") == false) this.map_leaflet.keyboard.disable();
+      // Disable dragging (also doubleClickZoom)
```

Comment:
I tend to comment all the things a made, probably too much:)
Comment a line is probably not a good idea but I think it is a good practice place one line comment each few blocks of code, not the code itself but the logic behind them.